src/LocalistProcessor.php

```diff
@@ -166,6 +174,30 @@ private function create_file_and_array($url) {
   }
 
 
+  private function create_media_from_file($image_array,$media_name) {
+    $media_id = $this->find_media_for_file($image_array);
+    if($media_id) {
+      return $media_id;
+    }
+
+    $media = Media::create([
+      'bundle'           => 'image',
+      'uid'              => 1,
+      'field_media_image' => $image_array,
+    ]);
+    $media->setName($media_name)->setPublished(TRUE)->save();
```

Comment:
Working on the news importer project, using this as an example -- code hints are telling me `setPublished()` does not accept arguments. (_[Relevant source code](https://git.drupalcode.org/project/drupal/-/blob/86d449bc46dbb838c4c51277689d964f58c20a41/core/lib/Drupal/Core/Entity/EntityPublishedTrait.php#L51-59)_)

Recommend:
```suggestion
    $media->setName($media_name)->setPublished()->save();
```